sympy/physics/units/tests/test_dimensions.py

```diff
@@ -65,17 +66,33 @@ def test_Dimension_properties():
 
 
 def test_Dimension_add_sub():
-    assert length + length == length
+    assert length + length == length + foot == foot + length == length
     assert length - length == length
     assert -length == length
 
-    raises(TypeError, lambda: length + 1)
-    raises(TypeError, lambda: length - 1)
     raises(ValueError, lambda: length + time)
```

Comment:
I thought that dimensions are not checked for consistency any more, unless specified, so `length + time` should not raise an error.

Reply:
hmm...good catch. Not intentionally left behind. I will look into it.